test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
The e2e tests in Kubernetes are:
* already in Ginkgo
* maintained by someone else
* a super set of the validation tests that have been ported from bash
* (are already all being run against ACS-Engine in a few different Jenkins jobs that are not used to gate PRs)

So why not just invoke the relevant small subset of the Kubernetes e2es instead of authoring our own copy to now maintain?